Tools/TestWebKitAPI/Tests/WebKitCocoa/WKHTTPCookieStore.mm

```diff
@@ -817,6 +817,47 @@ static bool areCookiesEqual(NSHTTPCookie *first, NSHTTPCookie *second)
     TestWebKitAPI::Util::run(&done);
 }
 
+#if __IPHONE_OS_VERSION_MIN_REQUIRED >= 180000 || __MAC_OS_X_VERSION_MIN_REQUIRED >= 150000
```

Comment:
What about tvOS, watchOS, visionOS?  This should probably be in PlatformHave.h instead.